packages/react/src/UnorderedList/UnorderedList.tsx

```diff
@@ -10,18 +10,27 @@ import { UnorderedListItem } from './UnorderedListItem'
 
 export type UnorderedListProps = {
   markers?: boolean
+  inverseColor?: boolean
 } & PropsWithChildren<HTMLAttributes<HTMLUListElement>>
 
 type UnorderedListComponent = {
   Item: typeof UnorderedListItem
 } & ForwardRefExoticComponent<UnorderedListProps & RefAttributes<HTMLUListElement>>
 
 export const UnorderedList = forwardRef(
-  ({ children, markers = true, className, ...restProps }: UnorderedListProps, ref: ForwardedRef<HTMLUListElement>) => {
+  (
+    { children, markers = true, inverseColor, className, ...restProps }: UnorderedListProps,
+    ref: ForwardedRef<HTMLUListElement>,
+  ) => {
     return (
       <ul
         ref={ref}
-        className={clsx('amsterdam-unordered-list', !markers && 'amsterdam-unordered-list--no-markers', className)}
+        className={clsx(
+          'amsterdam-unordered-list',
+          !markers && 'amsterdam-unordered-list--no-markers',
+          inverseColor && 'amsterdam-unordered-list--inverse-color',
```

Comment:
This is only a reorder.

```suggestion
          inverseColor && 'amsterdam-unordered-list--inverse-color',
          !markers && 'amsterdam-unordered-list--no-markers',
```